koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -945,3 +945,404 @@ def handler(self, src, target, opts=None):
             'repositories': all_repositories,
             'koji_builds': all_koji_builds,
         }
+
+
+class BuildSourceContainerTask(BaseTaskHandler):
+    # Start builds via osbs
+    Methods = ['buildSourceContainer']
+    # Same value as for regular 'build' method.
+    _taskWeight = 2.0
+
+    # JSON Schema definition for koji buildSourceContainer task parameters
+    # Used to validate arguments passed to the handler() method of this class
+    PARAMS_SCHEMA = {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "description": "Parameters for a koji buildSourceContainer task.",
+
+        "type": "array",
+        "items": [
+            {
+                "type": "string",
+                "description": "Build target."
+            },
+            {
+                "type": ["object", "null"],
+                "properties": {
+                    "scratch": {
+                        "type": "boolean",
+                        "description": "Perform a scratch build?"
+                    },
+                    "koji_build_id": {
+                        "type": ["string", "null"],
+                        "description": "Koji build id for sources"
+                    },
+                    "koji_build_nvr": {
+                        "type": ["string", "null"],
+                        "description": "Koji build nvr for sources"
+                    },
+                    "signing_intent": {
+                        "type": ["string", "null"],
+                        "description": "Signing intent of the ODCS composes."
+                    }
+                }
+            }
+        ]
```

Comment:
Also we want exactly two items in the array:
```
        "minItems": 2,
        "additionalItems": False
```
(this is probably also missing on the buildContainer schema)

Reply:
ok